src/elements/content-preview/ContentPreview.scss

```diff
@@ -75,3 +75,22 @@
 .bcpr.bcpr-thumbnails-open .bcpr-navigate-left {
     left: 226px;
 }
+
+.is-responsive-web {
+    @include breakpoint($medium-screen) {
+        .be.bcpr {
+            position: relative;
+        }
+
+        .bcpr {
+            .bcpr-body {
+                position: static;
```

Comment:
Is `position: static` needed?

For screen sizes between 700px (`.be-is-small`) and 768px, the position is `static` by default.

For screen sizes 700px and less (i.e. when `.be-is-small` is applied), `.bcpr-body` has `position: relative;` ([from this](https://github.com/box/box-ui-elements/blob/master/src/elements/content-preview/ContentPreview.scss#L63-L66)) but because `.bcpr` and `.bcpr-body` are the same sizes, this shouldn't have an effect.

Reply:
Alternatively, I think we could set `.bcpr-body` to `position: relative;` here and then remove these lines: https://github.com/box/box-ui-elements/blob/3b8e7767d3b88ec926e99c278f37843c1292ade1/src/elements/content-preview/ContentPreview.scss#L81-L83

This allows us in the future to remove this: https://github.com/box/box-ui-elements/blob/3b8e7767d3b88ec926e99c278f37843c1292ade1/src/elements/content-preview/ContentPreview.scss#L64-L66

Reply:
The main reason for applying `position: static` was to ensure the `position: absolute` from `aside` had a clear path to anchor on the `position relative` of 

`.be.bcpr { 
     position: relative; 
 } `

I tried the above and in Storybook it seems the Preview drawer doesn't fully cover the filename header when extended.

![localhost_6061__path=_story_elements-contentpreview--preview-with-annotations](https://user-images.githubusercontent.com/6400298/180033504-6a642429-7d91-4159-b63d-d7ebf1b0a4f3.png)